minvws/nl-kat-coordination
# Move event codes logging to KATalogus client

### Changes

Log event codes in the KATalogus client.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3904

### Demo


### QA notes

Please verify that we see logs still when we enable/disable, create/delete plugins including event codes!

---

### Code Checklist


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
